src/events/eventToAction.js

```diff
@@ -132,9 +132,28 @@ export default (state: PerAccountState, event: $FlowFixMe): EventAction | null =
         messageIds: event.message_ids ?? [event.message_id],
       };
 
+    case EventTypes.realm:
+      return {
+        type: EVENT,
+        event:
+          /* prettier-ignore */
+          event.op === 'update'
+            // Convert to an equivalent update_dict event, so reducers only have
+            // to handle update_dict.
```

Comment:
There's also that one property for which the `update` form has a hack where it provides a second property at the same time. If we're not handling that now (which we probably aren't), don't need to handle it in this change, but should leave a TODO comment to do so.